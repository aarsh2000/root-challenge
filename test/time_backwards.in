Driver Emmett
Trip Emmett 01:00 00:00 17.3
