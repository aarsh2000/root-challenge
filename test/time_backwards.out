Emmett: 0 miles
